src/execution_impl.rs

```diff
@@ -115,6 +147,7 @@ pub fn run_main_gc(
     param: &[u8],
     gas_costs: GasCosts,
 ) -> Result<GasCalibrationResult> {
+    // IMPORTANT TODO: consult how we'd like update this
```

Comment:
Seems like it should be handled in this PR no?

Reply:
Done